scheme/app/DoctrineMigrations/Version20190423104815.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20190423104815 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE Friends ADD t38Passthrough VARCHAR(255) DEFAULT \'no\' NOT NULL COMMENT \'[enum:yes|no]\'');
```

Comment:
Setting length to these field string enums definition in ORM files will avoid VARCHAR(255) in database fields